assets/src/web-stories-block/block/block-types/latest-stories/edit.js

```diff
@@ -67,11 +64,17 @@ const LatestStoriesEdit = ({ attributes, setAttributes }) => {
    *
    * @return {void}
    */
-  const fetchStories = async () => {
+  const fetchStories = useCallback(async (query) => {
```

Comment:
Why make this a callback?

Reply:
So it's memoized for use by the other hooks.